playwright/tests/rex-test/rex.behaviorspec.ts

```diff
@@ -235,3 +237,44 @@ test('note in content page', async ({ page, isMobile }) => {
   await BookPage.noteConfirmDialog(Actions.Save)
   expect(await BookPage.noteText()).toBe(editnoteText + noteText)
 })
+
+
+test('multiple note in content page', async ({ page, isMobile }) => {
+  test.skip(isMobile as boolean, 'test only desktop resolution')
+
+  // GIVEN: Open Rex page
+  const BookPage = new ContentPage(page)
+  const path = '/books/introduction-anthropology/pages/7-introduction'
+  await BookPage.open(path)
+
+  // AND: Signup as a new user
+  await rexUserSignup(page)
+  await expect(page).toHaveURL('/books/introduction-anthropology/pages/7-introduction')
+
+  // WHEN: Highlight 1 random paragraph
+  const paracount = BookPage.paracount()
+  const randomparanumber = randomNum(await paracount)
+  // const randomparanumber = 0
+  // // console.log(randomparanumber)
+  await BookPage.highlightText('green', randomparanumber)
+
+  const randomparanumber2 = randomNum(await paracount, randomparanumber)
+  await BookPage.highlightText('yellow', randomparanumber2)
+
+  // AND: Add note to the highlight and save
+
+  await BookPage.clickHighlight(1)
+  const noteText1 = randomstring()
+  await BookPage.addNote(noteText1)
+  // await BookPage.noteConfirmDialog(Actions.Save)
+  
+
+  
+  await BookPage.clickHighlight(0)
+  const noteText0 = randomstring()
+  await BookPage.addNote(noteText0)
+  expect(await BookPage.noteText(noteText1)).toContain(noteText1)
```

Comment:
@gregfitch can you please look at lines 276 & above. Does it feel like I the expected & actual parameters are the same. The actual value is defined in `content.page.ts` lines 231 - 241

Reply:
it's a bit redundant to expect a value in the object that you've just confirmed exists as the noteText function does its own search (if any text is returned by the function)

Reply:
@gregfitch i pushed some changes. when you get time can you please review if the note text verification make sense now.

PS. the pr is still draft